components/CaseStudiesList.vue

```diff
@@ -17,6 +17,25 @@
       </div>
     </div>
 
+    <div class="flex flex-wrap pt-12 pb-12 w-full lg:pr-4 lg:w-1/2">
+      <div class="w-full">
+        <h2 class="h-10 mb-2 w-auto">
+          <JebbitLogo class="h-10 w-auto" />
+        </h2>
+
+        <p class="lg:h-12">
+          Reworking the standard way clients create Jebbit campaigns to reduce complexity.
```

Comment:
Can we make this more generic? People may not know what a Jebbit campaign is. Maybe something like `Simplifying the way clients create their ad campaigns`? Not sure exactly on the language.